oletools/msodde.py

```diff
@@ -777,19 +786,22 @@ def control_symbol(self, matchobject):
         # required to handle control symbols such as '\\'
         # inject the symbol as-is in the text:
         # TODO: handle special symbols properly
-        self.current_destination.data += matchobject.group()[1]
+        self.current_destination.data += matchobject.group()[1:2]
 
 
+RTF_START = b'\x7b\x5c\x72\x74'   # == b'{\rt' but does not mess up auto-indent
```

Comment:
which auto-indent?

Reply:
The one in vim. 

But that was more of a side note I had hoped to be medium-funny. I can easliy live with the original code if you prefer it

Reply:
that's ok :-)